abuse_test.go

```diff
@@ -97,7 +97,14 @@ func TestAbuseListPaginateAndPassStatuses(t *testing.T) {
 	abuseApi := AbuseApi{}
 
 	ctx := context.Background()
-	response, err := abuseApi.List(ctx, 1, [2]string{"OPEN", "WAITING"})
+	status := []string{"OPEN", "WAITING"}
+	opts := AbuseListReportsOptions{
+		PaginationOptions: PaginationOptions{
+			Limit: Int(1),
+		},
+		Status: status,
```

Comment:
You can directly write `Status: []string{"OPEN", "WAITING"}`, there is no need to make it variable before

Reply:
Done